systems_cross_building/cross_building.rst

```diff
@@ -157,22 +157,41 @@ Windows to Raspberry PI
 
 .. code-block:: text
 
+    target_host=arm-linux-gnueabihf
+    standalone_toolchain=C:/sysgcc/raspberry
+    cc_compiler=gcc
+    cxx_compiler=g++
+
     [settings]
     os_build=Windows
     arch_build=x86_64
     os=Linux
-    arch=armv7hf
+    arch=armv7
```

Comment:
Actually, this would be armv6 for older RPI (maybe a note somewhere?)

Reply:
Right. Rpi 1 is armv6, it has to be updated in this table: http://docs.conan.io/en/latest/systems_cross_building/cross_building.html#arm-architecture-reference